modules/amp_dfp/amp_dfp.module

```diff
@@ -122,6 +134,16 @@ function amp_dfp_preprocess_amp_dfp_tag(&$variables) {
 
   $to_json = array('targeting' => $targeting);
   $variables['amp_ad_json'] = drupal_json_encode($to_json);
+  $variables['rtc_config'] = '';
+  if(isset($tag->settings['rtc_config']) && !is_null($tag->settings['rtc_config']['timeout'])) {
+    $rtc_config = new RealTimeConfiguration(
+        $tag->settings['rtc_config']['vendor'],
+        $tag->settings['rtc_config']['urls'],
+        $tag->settings['rtc_config']['timeout']
+      );
+
+    $variables['rtc_config'] = drupal_json_encode($rtc_config->jsonSerialize());
```

Comment:
Shouldn’t need to call the interface method here - json_encode() should be enough - see 
http://php.net/manual/en/class.jsonserializable.php